test/core/sdk.spec.js

```diff
@@ -392,9 +392,9 @@ describe('SDK', () => {
           .then(() => {
             expect(sdk.client.request).toBeCalledWith(
               'https://api.uphold.com/v0/foo',
-              'get',
```

Comment:
Why did we removed the `get` assert?

Reply:
The request `method` is not an argument anymore. It is now provided by the `options` object.

Reply:
So, we need to create new tests to cover this scenario.

Reply:
Added on a different file.